client/dashboard/sites/backups/style.scss

```diff
@@ -1,7 +1,11 @@
 .dashboard-backups__list-icon {
-    position: relative;
-    top: 50%;
-    inset-inline-start: 50%;
-    transform: translate(-50%, -50%);
-    fill: $gray-700;
+	position: relative;
+	top: 50%;
+	inset-inline-start: 50%;
+	transform: translate(-50%, -50%);
+	fill: $gray-700;
+}
+
+.dashboard-backups__progress-bar{
+	--wp-components-color-foreground: var(--wp-admin-theme-color);
```

Comment:
I'm placing this here since we will require it on the backup in progress state and the download progress page :)